src/plugins/dashboard/public/dashboard_listing/dashboard_listing.tsx

```diff
@@ -225,35 +254,56 @@ export const DashboardListing = ({
           FormattedRelative,
         }}
       >
-        <TableListView<DashboardSavedObjectUserContent>
-          getDetailViewLink={({ id, attributes: { timeRestore } }) =>
-            getDashboardUrl(id, timeRestore)
-          }
-          deleteItems={!showWriteControls ? undefined : deleteItems}
-          createItem={!showWriteControls ? undefined : createItem}
-          editItem={!showWriteControls ? undefined : editItem}
-          entityNamePlural={getEntityNamePlural()}
-          title={getTableListTitle()}
-          headingId="dashboardListingHeading"
-          initialPageSize={initialPageSize}
-          initialFilter={initialFilter}
-          entityName={getEntityName()}
-          listingLimit={listingLimit}
-          emptyPrompt={emptyPrompt}
-          findItems={fetchItems}
-          id="dashboard"
-        >
-          <>
-            {children}
-            <DashboardUnsavedListing
-              goToDashboard={goToDashboard}
-              unsavedDashboardIds={unsavedDashboardIds}
-              refreshUnsavedDashboards={() =>
-                setUnsavedDashboardIds(dashboardSessionStorage.getDashboardIdsWithUnsavedChanges())
-              }
+        <PageTemplate panelled data-test-subj={pageDataTestSubject}>
+          {withPageTemplateHeader && (
+            <KibanaPageTemplate.Header
+              pageTitle={<span id={HEADING_ID}>{title}</span>}
+              data-test-subj="top-nav"
             />
-          </>
-        </TableListView>
+          )}
+          <KibanaPageTemplate.Section
+            aria-labelledby={hasInitialFetchReturned ? HEADING_ID : undefined}
+            restrictWidth={restrictPageSectionWidth}
+            paddingSize={pageSectionPadding}
+            data-test-subj="dashboard-section"
+          >
+            <>
+              {children}
+              <DashboardUnsavedListing
+                goToDashboard={goToDashboard}
+                unsavedDashboardIds={unsavedDashboardIds}
+                refreshUnsavedDashboards={() =>
+                  setUnsavedDashboardIds(
+                    dashboardSessionStorage.getDashboardIdsWithUnsavedChanges()
+                  )
+                }
+              />
+              <TableListViewTable
```

Comment:
Shared UX team prefer not to add more props to `TableListView`, so suggested us to use `TableListViewTable` to avoid adding more new props.